sql/moz-fx-data-marketing-prod/ga_derived/firefox_whatsnew_summary_v2/query.sql

```diff
@@ -0,0 +1,42 @@
+WITH wnp_visits AS (
+  SELECT
+    date,
+    visit_identifier,
+    TRIM(page_path_level1, '/') AS locale,
+    page_level_2 AS version,
+    mozfun.norm.browser_version_info(page_level_2) AS version_info,
+    country,
+    CAST(bounces AS boolean) AS is_bounce
+  FROM
+    `moz-fx-data-marketing-prod.ga_derived.www_site_hits_v2`
+  WHERE
+    date = @submission_date
+    AND hit_type = 'PAGE'
+    -- Match page paths like "/{locale}/firefox/{version}/whatsnew/..."
+    -- Version regular expression is adapted from https://github.com/mozilla/bedrock/blob/main/bedrock/releasenotes/__init__.py
+    AND page_level_1 = 'firefox'
```

Comment:
What is the difference between `page_path_level1` and `page_level_1`?

Reply:
Basically page path level 1 is like the first offset, and then page level 1 is the second offset (don't ask me why they named it that way, that's how the old tables were LOL)
Page level 1 = SPLIT(REGEXP_REPLACE(page_location, '^https://www.mozilla.org', ''), '/')[
      SAFE_OFFSET(2)
    ] AS page_level_1

Page path level 1 = SPLIT(REGEXP_REPLACE(all_events.page_location, '^https://www.mozilla.org', ''), '/')[
      SAFE_OFFSET(1)
    ]